posthog/queries/trends.py

```diff
@@ -97,7 +97,7 @@ def build_dataarray(
 
     # following finds top 20 breakdown in given queryset then removes other rows from data array
     if len(cohort_keys) > 20:
-        top20keys = [x[0] for x in sorted(cohort_dict.items(), key=lambda x: -x[1])[:20]]
+        top20keys = [x[0] for x in sorted(cohort_dict.items(), key=lambda x: -x[1])[:25]]
```

Comment:
I think the behavior change here is intended, perhaps just need to update the comment above and variable name?

Reply:
yup